### PR TITLE
PWN-3763 - Add unit tests and ktlint to github actions

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3763

## Description of Work

Setup first jobs of Github action

### CODE REVIEW

File: .github/workflows/android_lint.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Android Kotlin lint
+
+on: [ pull_request]
```
Comment (1sakov) [inline .github/workflows/android_lint.yml:3]: We should run linter and unit tests after merging in develop/main branches as well

Comment (davran1001) [reply .github/workflows/android_lint.yml:3]: i will add this steps on merge into develop branch

**Review by kuznetsov — APPROVED** [review]

File: .github/workflows/android_unit_test.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with: { java-version: 11 }
+
+      - name: Setup google-json from secrets
+        env:
+          FIREBASE_SECRET: ${{ secrets.GOOGLE_SERVICE_JSON_SECRET }}
+        run: echo $FIREBASE_SECRET > app/google-services.json
```
Comment (gslevinkov) [inline .github/workflows/android_unit_test.yml:24]: is it hex of the secret or just the raw file?

Comment (davran1001) [reply .github/workflows/android_unit_test.yml:24]: raw file

**Review by kabramovp2p — APPROVED** [review]

**Pull request merged**